rest/endpoints/user_configuration/user_preferences.yml

```diff
@@ -107,3 +107,67 @@ paths:
       responses:
         204:
           description: Successful response, the requested reset password email was sent. No content returned. Also returned if the user email does not exist.
+
+
+    # -------------------------------------------------------------------------------------------------------------------------------------
+    # CONFIRM PASSWORD RESET
+    # -------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
I think these comment should just separate the different sections. The sections are denoted per endpoint by tags. This endpoint is tagged as "Password Reset"